tools/boards.txt.py

```diff
@@ -652,9 +643,9 @@ def all_flash_size ():
 		macrolist += board['macro']
 	macrolist += [ 'debug_menu' ]
 	for block in macrolist:
-		for keyval in macros[block]:
-			if not ('opts' in board) or not (keyval[0] in board['opts']):
-				print short + keyval[0] + '=' + keyval[1]
+		for key in macros[block]:
+			if not ('opts' in board) or not (str(key) in board['opts']):
```

Comment:
I also think the dict makes this line clearer with 'key' instead of 'keyval[0]' :)